addon/-private/engine-ext.js

```diff
@@ -10,6 +11,7 @@ Engine.reopen({
     if (!(this instanceof Application)) {
       registry.register('component:link-to', EngineScopedLinkComponent);
       registry.register('component:link-to-external', ExternalLinkComponent);
+      registry.register('service:engineRouterService', EngineRouterService);
```

Comment:
We should not need to register under another name here, we should register `service:router` (not a different name)